client/login/magic-login/emailed-login-link-expired.jsx

```diff
@@ -20,7 +20,8 @@ import RedirectWhenLoggedIn from 'components/redirect-when-logged-in';
 import { hideMagicLoginRequestForm } from 'state/login/magic-login/actions';
 import { recordPageView } from 'state/analytics/actions';
 
-const nativeLoginUrl = login( { isNative: true } );
+const nativeLoginUrl = login( { isNative: true, twoFactorAuthType: 'link' } );
```

Comment:
Maybe lets add a comment here to explain that we don't have actually a two factor authentication type `link` and it's just to build a url?

Reply:
I see your point, but it's not the only place where this parameter has been used like this. The right way to address that particular issue would be to actually rename that `twoFactorAuthType` parameter into `flow` or something like that everywhere - obviously in a different pull request.

Reply:
I actually went and looked what's that new `twoFactorAuthType === 'link'`